addons/sourcemod/scripting/rngfix.sp

```diff
@@ -112,6 +118,13 @@ void DebugLaser(int client, const float p1[3], const float p2[3], float life, fl
 public APLRes AskPluginLoad2(Handle myself, bool late, char[] error, int err_max)
 {
    g_bLateLoad = late;
+   RegPluginLibrary("rngfix");
+
+   g_hOnTriggerDetected = CreateGlobalForward("RNGFix_OnTriggerDetected", ET_Ignore, Param_Cell, Param_String);
```

Comment:
why do we need these forwards

Reply:
We can use them instead of making repeating timer for hooking them.
Exemple: 
https://github.com/srcdslab/sm-plugin-ButtonNotifier/blob/master/addons/sourcemod/scripting/ButtonNotifier.sp#L60

Code usage exemple: 
```
public void RNGFix_OnTriggerStartTouch(int entity, int other)
{
	if (entity >= 2048) return;

	// Debug
	PrintToServer("OnTriggerStartTouch - entity: %d - Activator: %L", entity, other);

	char classname[64];
	GetEntityClassname(entity, classname, sizeof(classname));
	if (strcmp(classname,"trigger_push", false) == 0 || strcmp(classname,"trigger_once", false) == 0)
		// Do stuff
}
```

Reply:
yeah but that creates also unnecessary dependencies because rngfix is only a fix and not really supposed to expose forwards

Reply:
Alright.